### PR TITLE
Add support for release/5.7 branch

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci smoke test and merge

File: utils/update_checkout/update-checkout-config.json [hunk 1]
```diff
@@ -160,6 +160,48 @@
                 "swift-experimental-string-processing": "swift/main"
             }
         },
+        "release/5.6": {
```
Comment (buttaface) [inline utils/update_checkout/update-checkout-config.json:163]: Missed one.

Comment (shahmishal) [inline utils/update_checkout/update-checkout-config.json:163]:
```suggestion
        "release/5.7": {
```

**Comment by shahmishal** [comment]
@swift-ci smoke test and merge

**Pull request merged**